src/v/storage/probe.cc

```diff
@@ -70,87 +70,108 @@ void probe::setup_metrics(const model::ntp& ntp) {
-        sm::make_counter(
-          "batches_written",
-          [this] { return _batches_written; },
-          sm::description("Total number of batches written"),
-          labels),
-        sm::make_total_bytes(
-          "read_bytes",
-          [this] { return _bytes_read; },
-          sm::description("Total number of bytes read"),
-          labels),
-        sm::make_total_bytes(
-          "cached_read_bytes",
-          [this] { return _cached_bytes_read; },
-          sm::description("Total number of cached bytes read"),
-          labels),
-        sm::make_counter(
-          "batches_read",
-          [this] { return _batches_read; },
-          sm::description("Total number of batches read"),
-          labels),
-        sm::make_counter(
-          "cached_batches_read",
-          [this] { return _cached_batches_read; },
-          sm::description("Total number of cached batches read"),
-          labels),
-        sm::make_counter(
-          "log_segments_created",
-          [this] { return _log_segments_created; },
-          sm::description("Number of created log segments"),
-          labels),
-        sm::make_counter(
-          "log_segments_removed",
-          [this] { return _log_segments_removed; },
-          sm::description("Number of removed log segments"),
-          labels),
-        sm::make_counter(
-          "log_segments_active",
-          [this] { return _log_segments_active; },
-          sm::description("Number of active log segments"),
-          labels),
-        sm::make_counter(
-          "batch_parse_errors",
-          [this] { return _batch_parse_errors; },
-          sm::description("Number of batch parsing (reading) errors"),
-          labels),
-        sm::make_counter(
-          "batch_write_errors",
-          [this] { return _batch_write_errors; },
-          sm::description("Number of batch write errors"),
-          labels),
-        sm::make_counter(
-          "corrupted_compaction_indices",
-          [this] { return _corrupted_compaction_index; },
-          sm::description("Number of times we had to re-construct the "
-                          ".compaction index on a segment"),
-          labels),
-        sm::make_counter(
-          "compacted_segment",
-          [this] { return _segment_compacted; },
-          sm::description("Number of compacted segments"),
-          labels),
-        sm::make_gauge(
-          "partition_size",
-          [this] { return _partition_bytes; },
-          sm::description("Current size of partition in bytes"),
-          labels),
-        sm::make_total_bytes(
-          "compaction_ratio",
-          [this] { return _compaction_ratio; },
-          sm::description("Average segment compaction ratio"),
-          labels),
-      });
+      {sm::make_total_bytes(
+         "written_bytes",
+         [this] { return _bytes_written; },
+         sm::description("Total number of bytes written"),
+         labels,
+         sm::impl::shard())
```

Comment:
These shard parameters are no longer needed (they're defaulted, and were added since aggregate_labels were added as the last param)

Reply:
Good spot. Removed them.